src/modules/fw_pos_control_l1/FixedwingPositionControl.cpp

```diff
@@ -2588,8 +2568,8 @@ FixedwingPositionControl::tecs_update_pitch_throttle(const hrt_abstime &now, flo
 		bool climbout_mode, float climbout_pitch_min_rad,
 		bool disable_underspeed_detection, float hgt_rate_sp)
 {
-	const float dt = math::constrain((now - _last_tecs_update) * 1e-6f, MIN_AUTO_TIMESTEP, MAX_AUTO_TIMESTEP);
-	_last_tecs_update = now;
+	const float dt = math::constrain((now - _time_last_tecs_update) * 1e-6f, MIN_AUTO_TIMESTEP, MAX_AUTO_TIMESTEP);
```

Comment:
Is there a reason that this needs to differ from the `control_interval` ? If not, I think we actually don't need to pass `now` all the way down here and remove `_last_tecs_update`

Reply:
yeah i also agree we probably dont need two separate intervals.. as they are called at the same rate anyway.